cli/formatter/formatter.go

```diff
@@ -0,0 +1,57 @@
+package formatter
+
+import (
+	"sort"
+	"strings"
+)
+
+//Format key to output
+const (
+	TableFormat = "table"
```

Comment:
The difference between table and raw is that raw's output includes the container header, right?

Reply:
yes, if using table, the first line should be the container header. like this:
pouch ps -a --format "table {{.ID}}\t{{.Status}}\t{{.Created}}\t{{.Name}}"
ID   Status   Created   Name
f78484   Up 32 seconds   33 seconds ago   f78484
58b7de   Up 3 days   3 days ago   58b7de

if using raw, result like below:
pouch ps -a --format "raw"
Name:f78484
ID:f78484
Status:Up 13 seconds
Created:13 seconds ago
Image:registry.hub.docker.com/library/ubuntu:latest
Runtime:runc

Name:58b7de
ID:58b7de
Status:Up 3 days
Created:3 days ago
Image:registry.hub.docker.com/library/ubuntu:latest
Runtime:runc